### DISCUSSION TITLE
music/rock/

### DISCUSSION THREAD

**Discussion by giscus** [KBVE/kbve#1644] [KBVE]
# music/rock/

Rock music, a genre marked by powerful backbeats and electric guitars, has shaped popular music since the 1950s.
From legendary artists like The Beatles and Led Zeppelin to diverse subgenres like punk, hard rock, and alternative, Rock continues to influence and inspire.

https://kbve.com/music/rock/

**Comment by h0lybyte** [comment]
Need more rock songs!
